cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -265,6 +265,53 @@
                     {{ additional_check.statement }}
                 </td>
             </tr>
+     {% elif additional_check.type == 'inconsistent_schema_version_used' and additional_check.statement_type == 'person' %}
+            <tr>
+                <td>
+                    {% blocktrans %}This Person Statement is a different version of the schema than the file.{%endblocktrans%}
```

Comment:
Could these error messages all be edited to echo the language used in explore.html:

This ??? Statement and the first statement of the submitted data reference different BODS versions.

Reply:
Done